src/Utils.ps1

```diff
@@ -176,6 +176,14 @@ function Add-PoshGitToProfile {
         $profileContent = "`nImport-Module '$ModuleBasePath\posh-git.psd1'"
     }
 
+    # Make sure the profile path exists
+    $profileDir = Split-Path -Parent $profilePath
+    if (-not (Test-Path -LiteralPath $profileDir)) { 
```

Comment:
In the code base we tend not to use `-not` in favor of `!`.  We're not 100% there yet but I don't want to introduce more script that needs to be changed.